test_lib.go

```diff
@@ -45,17 +44,8 @@ func getTestCfg(t *testing.T, b *testing.B) *config.Config {
 		panic(err)
 	}
 
-	cfg.DatabaseDSN = "test.db"
-
-	if t != nil {
-		cfg.DatabaseDSN = path.Join(t.TempDir(), "test.db")
-	}
-
-	if b != nil {
-		cfg.DatabaseDSN = path.Join(b.TempDir(), "test.db")
-	}
-
-	cfg.DatabaseType = "sqlite"
```

Comment:
`sqlite` is not supporting multiple threads to access the db at the same time. Switching to use `postgres` to integration test.